### ISSUE TITLE
Display who left a reaction

### ISSUE THREAD

**Issue by Seirdy** [feditext/feditext#296]
**Is your feature request related to a problem? Please describe.**
When viewing a post with reactions, Feditext does not display the sender of each reaction.

**Describe the solution you'd like**

A clickable counter for reactions alongside favorites. Clicking it should open something resembling Firefish or IceShrimp's reaction information, showing a tab strip of received reactions and a list of users for the currently active tab.

**Describe alternatives you've considered**
Scrolling through notifications to piece together who sent reactions (although this is also broken; see #292).

**Comment by VyrCossont** [comment]
This should be possible in Akkoma and [Firefish](https://codeberg.org/firefish/firefish/src/branch/develop/packages/megalodon/src/entities/reaction.ts#L9)/Iceshrimp, but not [Glitch](https://github.com/glitch-soc/mastodon/pull/2221).

**Comment by ShadowJonathan** [comment]
Glitch might soon support reactions; https://github.com/glitch-soc/mastodon/pull/2462

**Comment by TheEssem** [comment]
> Glitch might soon support reactions; [glitch-soc/mastodon#2462](https://github.com/glitch-soc/mastodon/pull/2462)

Yes, but there's still no API for checking who reacted to a post with what. I started on this in another branch (TheEssem/mastodon#11), but it's still somewhat incomplete and I intend to send it to glitch-soc after the main reactions PR (and I'm mainly looking for review before I merge it into my own fork).

**Comment by VyrCossont** [comment]
@TheEssem mind pinging me when it's reviewable? If it's compatible with the Akkoma and Iceshrimp APIs for the same thing, that'd help.
